nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractSyslogProcessor.java

```diff
@@ -29,34 +29,28 @@
     public static final AllowableValue TCP_VALUE = new AllowableValue("TCP", "TCP");
     public static final AllowableValue UDP_VALUE = new AllowableValue("UDP", "UDP");
 
-    public static final PropertyDescriptor PROTOCOL = new PropertyDescriptor
+    public static final PropertyDescriptor PROTOCOL_PROP = new PropertyDescriptor
             .Builder().name("Protocol")
             .description("The protocol for Syslog communication.")
             .required(true)
             .allowableValues(TCP_VALUE, UDP_VALUE)
             .defaultValue(UDP_VALUE.getValue())
             .build();
-    public static final PropertyDescriptor PORT = new PropertyDescriptor
+    public static final PropertyDescriptor.Builder PORT_PROP_BUILDER = new PropertyDescriptor
```

Comment:
Any reason not to just update the property descriptors directly in AbstractSyslogProcessor?

Assuming we don't support EL for protocol, and given the timeout property isn't used by ListenSyslog, we'd only have to modify two additional lines in ListenSyslog where it evaluates the port and charset.

Reply:
Yeah, the reason was just that I was shying away from modifying the ListenSyslog processor as well but that's probably a better way to go. Will fix